adapters/amqp-vertx-quarkus/src/main/resources/application.properties

```diff
@@ -1,4 +1,10 @@
+quarkus.http.port=${vertx.health.port}
 # this needs to be configured at build time in order for native images to be able to log at TRACE level
 quarkus.log.min-level=TRACE
+quarkus.micrometer.binder-enabled-default= false
+quarkus.micrometer.binder.jvm=true
+quarkus.micrometer.binder.kafka.enabled=true
```

Comment:
I think this currently needs to be set to `false` as we are already registering Kafka clients explicitly using the `MicrometerKafkaClientMetricsSupport` class.

Reply:
I wasn't sure about that so it's good that you mention it. This applies to all components then, right?

Reply:
Yes.